ketcher-autotests/tests/Structure-Creating-&-Editing/SMARTS-attributes/Atom-properties-attributes/atom-properties-attributes.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { Page, test } from '@playwright/test';
+import {
+  BondTypeName,
+  clickInTheMiddleOfTheScreen,
+  doubleClickOnAtom,
+  pressButton,
+  selectBond,
+  takeEditorScreenshot,
+  waitForAtomPropsModal,
+  waitForPageInit,
+} from '@utils';
+import {
+  checkSmartsValue,
+  setAtomicMass,
+  setCharge,
+  setLabel,
+  setValence,
+} from '../utils';
+
+async function drawStructure(page: Page) {
+  await selectBond(BondTypeName.Single, page);
```

Comment:
maybe we can refactore this function for improved code readability and maintainability:

async function drawStructure(page: Page, numberOfClicks: number) {
  await selectBond(BondTypeName.Single, page);
  for (let i = 0; i < numberOfClicks; i++) {
    await clickInTheMiddleOfTheScreen(page);
  }
}

await drawStructure(page, 3);

Reply:
Done :)